templates/sabayon.json

```diff
@@ -1,16 +1,15 @@
 {
     "create_message": ["You just created a Sabayon container ",
-                       "(release=RELEASE, arch=ARCH, variant=VARIANT)\n\n",
-                       "To enable sshd, run: rc-update add sshd"],
+                       "(release=RELEASE, arch=ARCH, u=Unprivileged Container)\n\n",
```

Comment:
Ah, then the image we build on Jenkins should always use "-u" so that the resulting image can be used for both privileged and unprivileged containers.

If someone cares about the bits that only work in privileged containers, they should just use the lxc-sabayon template directly.